core/modules/czar/python/proxy.py

```diff
@@ -50,9 +63,11 @@ class Lock:
     requests when it is waiting on mysql queries that it knowingly
     initiates in its C-layer, whereas it treats its Lua plugin code as a
     single-threaded black-box."""
-    createTmpl = "CREATE TABLE IF NOT EXISTS %s (chunkId INT, code SMALLINT, message CHAR(255), timeStamp FLOAT) ENGINE=MEMORY;"
+    createTmpl = "CREATE TABLE IF NOT EXISTS %s " \
+                 "(chunkId INT, code SMALLINT, message CHAR(255), " \
```

Comment:
Storing messages inside TEXT might be resource consuming, isn't it? If yes, we may improve it later, depending on our user experience on log/error message length on the mysql proxy/client?